plenum/common/ledger_manager.py

```diff
@@ -86,6 +88,29 @@ def addLedger(self, iD: int, ledger: Ledger,
             verifier=MerkleVerifier(ledger.hasher)
         )
 
+    def _cancel_request_ledger_statuses_and_consistency_proofs(self, ledger_id):
+        if ledger_id in self.request_ledger_status_action_ids:
+            self._cancel(self.request_ledger_status_action_ids[ledger_id])
+            self.request_ledger_status_action_ids[ledger_id] = None
+
+        if ledger_id in self.consistency_proof_action_ids:
+            self._cancel(self.consistency_proof_action_ids[ledger_id])
+            self.consistency_proof_action_ids[ledger_id] = None
+
+    def request_ledger_status_if_needed(self, ledger_id):
```

Comment:
The suffix `_if_needed` in the method name seems to be redundant because cancellation of the scheduled action is not a responsibility of this method itself.
It would be better to name this method `reask_for_ledger_status`.

Reply:
Done.